runtime/Java/src/org/antlr/v4/runtime/atn/ATNDeserializer.java

```diff
@@ -160,6 +160,7 @@ static protected boolean isFeatureSupported(UUID feature, UUID actualUuid) {
 	@SuppressWarnings("deprecation")
 	public ATN deserialize(char[] data) {
 		data = data.clone();
+		System.gc(); // presumably helps free up deserialized ATN
```

Comment:
It's not recommended to call `gc` manually, it may decrease performance. JVM better detects the best moment when gc should be called.